src/NHibernate/Proxy/AbstractProxyFactory.cs

```diff
@@ -18,7 +19,7 @@ public abstract class AbstractProxyFactory: IProxyFactory
 		protected virtual MethodInfo GetIdentifierMethod { get; private set; }
 		protected virtual MethodInfo SetIdentifierMethod { get; private set; }
 		protected virtual IAbstractComponentType ComponentIdType { get; private set; }
-		protected virtual bool OverridesEquals { get; set; }
+		protected virtual bool InterceptsEquals { get; set; }
```

Comment:
You cannot just rename the field.

Reply:
Yes of course, I forgot about this being a breaking change.